lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KBVGateway.java

```diff
@@ -22,6 +23,7 @@
 import java.util.Objects;
 
 public class KBVGateway {
+    public static final String EXPERIAN_IIQ_REQUEST = "experian_iiq_request";
```

Comment:
This variable can be `private`
Is there any value in tweaking the name slightly ?
e.g. `"experian_iiq_endpoint"` or `"experian_iiq_request_type"`